demos/mqtt/mqtt_demo_plaintext/mqtt_demo_plaintext.c

```diff
@@ -288,6 +288,7 @@ int main( int argc, char ** argv )
     int status;
     MQTTContext_t context;
 
+    /* Establish TCP connection and MQTT session. */
```

Comment:
Would you please update this comment too as mentioned by Amit above.

Reply:
Updated